instrumentation/jedis-5.0.0/src/main/java/redis/clients/jedis/Connection_Instrumentation.java

```diff
@@ -0,0 +1,90 @@
+/*
+ *
+ *  * Copyright 2020 New Relic Corporation. All rights reserved.
```

Comment:
If this is a net new instrumentation module then the year should be 2024.

The other file (`JedisPubSubBase_Instrumentation.java`) also needs a copyright header.

Reply:
Updated!